### ISSUE TITLE
setup function relies on reading cross-origin style sheets (CSSOM spec violation)

### ISSUE THREAD

**Issue by miketaylr** [sampotts/rangetouch#8]
https://github.com/sampotts/rangetouch/blob/46e6b12f196bc6511d44acbaf6947633bd4ddb41/src/js/rangetouch.js#L52-L54

This violates the CSSOM spec:

```
The insertRule(rule, index) method must run the following steps:

    If the origin-clean flag is unset, throw a SecurityError exception.
    Return the result of invoking insert a CSS rule rule in the CSS rules at index. 
```

https://drafts.csswg.org/cssom/#dom-cssstylesheet-insertrule

And breaks in Firefox (which follows the spec).

**Comment by miketaylr** [comment]
To workaround this, you could create a new stylesheet with this rule and append that to the document. Or just use inline styles?

**Comment by sampotts** [comment]
This is fixed in `v1.0.6` but I've also just released a complete rewrite which is in beta - `v2.0.0-beta.1`. It is now using a proper ES6 class so must be setup per input. There's also a static `RangeTouch.setup()` method that allows you to setup multiple inputs and returns an array of the references to the instances so you can destroy them etc. I think this will improve support for frameworks like React where you can import the lib and setup RangeTouch in your component. Give it a go and give feedback if you can. I'm still testing myself (hence beta) so there may be bugs.
